event_detection/replay_detector.py

```diff
@@ -0,0 +1,31 @@
+
+class ReplayDetector:
+
+    def __init__(self):
+       # the number of points expected to be scored before a replay happens
+       self.expected_points = 1
+       self.replay_frames = 0
+    
+    def check_game_status(self, data, model):
+        results = model.predict(data.reshape(1, -1))
+        print(results)
```

Comment:
Do we want to be printing to console? Seems like maybe a debug thing that should be cleaned up before the expo?